source/architecture.rst

```diff
@@ -117,15 +117,15 @@ Otherwise this is the standard GEMM C++ interface.
       buffer<double, 1> b{B, range<1>{N*P}};
       buffer<double, 1> c{C, range<1>{M*P}};
 
-      mkl::transpose nT = mkl::transpose::nontrans;
+      onemath::transpose nT = onemath::transpose::nontrans;
```

Comment:
Wait, did we decide on `oneapi::math::` namespace or `oneapi::onemath::` namespace ?  I thought it was the former.

Reply:
Thank you for your comment @spencerpatty . You are right the namespace is `oneapi::math::` in the renaming something went wrong I mistook some occurrences. I update them in [faf0bcc](https://github.com/uxlfoundation/oneAPI-spec/pull/596/commits/faf0bcc983f06c07ca3e21bef784c191409dcdf8)